README.md

```diff
@@ -1,27 +1,168 @@
-# Role Name
+# Gitlab
 
-A brief description of the role goes here.
+This role configures and installs a gitlab instance.
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
-
+This role depends on (postgres)[https://github.com/stuvusIT/postgresql], (redis)[https://github.com/stuvusIT/redis] and (nginx)[https://github.com/stuvusIT/nginx]
 
 ## Role Variables
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role.
-Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+Since Gitlab is higly configureable see the (gitlab example config file)[https://gitlab.com/gitlab-org/gitlab-ce/raw/f441fe7b548fd9cb87eb2f0eadfa88b2e312b692/config/gitlab.yml.example] 
+All variable names are gitlab_*. See below for some examples.
+
+| Variable                           | Default / Mandatory                              | Description                                                                                                                                                     |
+|------------------------------------|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| gitlab_host                   | `localhost`                                   | URL where gitlab is running. |
+| gitlab_port | `80` | Default port where gitlab ist listening |
+| gitlab_https | `false` | Enable/disable https. |
```

Comment:
So this enables https. Where do I have to place my key and cert?

Reply:
Below are some examples not every var